packages/web-components/src/components/ic-toggle-button-group/ic-toggle-button-group.stories.mdx

```diff
@@ -0,0 +1,447 @@
+import {
+  Meta,
+  Story,
+  Canvas,
+  ArgsTable,
+  Description,
+} from "@storybook/addon-docs";
+import { html } from "lit-html";
+import readme from "./readme.md";
+
+<Meta
+  title="Web Components/Toggle Button Group"
+  component="Toggle Button Group"
+/>
+
+<Description markdown={readme} />
+
+### Select type
+
+<Canvas>
+  <Story name="Select type" parameters={{ loki: { skip: true } }}>
+    {(args) =>
+      html`
+        <span>Single and manual</span>
+        <ic-toggle-button-group
+          select-type="single"
+          label="Label for single and manual"
+          accessible-label="accessible label for the toggle button group"
+        >
+          <ic-toggle-button label="First toggle"></ic-toggle-button>
+          <ic-toggle-button label="Second toggle"></ic-toggle-button>
+          <ic-toggle-button label="Third toggle"></ic-toggle-button>
+        </ic-toggle-button-group>
+        <br />
+        <span>Single and auto</span>
+        <ic-toggle-button-group
+          label="Label for single and manual"
```

Comment:
What is this label for? It's not unique (it says "single and manual" for all three groups) and isn't visible or read out by a screen reader (in my case, Voiceover)

Could it be removed?